### PR TITLE
OBDS-132 Added Progress Screen for loading demo data

### PR DESCRIPTION
**Pull request by DaRoTP**

### CODE REVIEW

File: src/js/components/locations-configuration/LoadDataPage.jsx [hunk 1]
```diff
@@ -90,31 +97,72 @@ const LoadDataContent = ({
       {ReactHtmlParser(summaryItemsList)}
     </div>
     <div className="d-flex justify-content-between m-3">
-      <button type="button" onClick={stepBackHandler} className="btn btn-outline-primary">
+      <button type="button" onClick={() => nextStep(null)} className="btn btn-outline-primary">
```
Comment (awalkowiak) [inline src/js/components/locations-configuration/LoadDataPage.jsx:100]: Add a `prevStep` handler. (Bring back `stepBackHandler` but as a `prevStep`). Unless you want to unify it under one handler, then call it `goToStep`.

File: src/js/components/locations-configuration/LoadDataPage.jsx [hunk 2]
```diff
@@ -90,31 +97,72 @@ const LoadDataContent = ({
...
       </button>
     </div>
   </React.Fragment>
 );
 
+const LoadDataProgressScreen = ({ nextStep }) => {
+  useEffect(() => {
+    apiClient.get('/openboxes/api/config/data/demo')
+      .then(() => {
+        nextStep(LOAD_DATA_STEPS.dataLoadedSuccess);
```
Comment (awalkowiak) [inline src/js/components/locations-configuration/LoadDataPage.jsx:114]: I don't like the idea of `dataLoadedSuccess` being its own step, it does not feel right to me. I'd prefer it to be content that is displayed in `then` and the same for error content being displayed in `catch`.

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: src/js/components/locations-configuration/LoadDataPage.jsx (hunk 1 above)
Comment (DaRoTP) [reply src/js/components/locations-configuration/LoadDataPage.jsx]: yes, I wanted to unify it as one handler and I agree **goToStep** make a lot more sens in that case than **nextStep**

**Review by awalkowiak — CHANGES_REQUESTED** [review]
@DaRoTP what I realized is that this load data page should be in a separate directory. Please, move this from `locations-configuration` to the `load-demo-data`. On top of that please extract every stateful component to separate files.

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**